src/components/organisms/settings/clientConfiguration.tsx

```diff
@@ -25,72 +25,67 @@ export const ClientConfiguration = () => {
-								className="flex-1 rounded-xl hover:border-gray-1100"
-								disabled
-								label={t("getToken.copyInputAriaLabel")}
-								value={token}
-							/>
-
-							<CopyButton
-								className="ml-4 rounded-md bg-gray-900 px-3 hover:bg-gray-950"
-								successMessage={t("getToken.copySuccess")}
-								text={token}
-							/>
-						</div>
-					) : (
-						<Button
-							className="border border-white px-3 font-medium text-white hover:bg-gray-1300"
-							onClick={createToken}
-							variant="filled"
-						>
-							<div className="flex items-center">
-								{!isLoading ? (
-									<IconSvg alt="New Project" className="fill-white" size="md" src={NewProject} />
-								) : null}
-
-								{isLoading ? <Loader size="sm" /> : null}
-							</div>
-
-							<div className="mr-1 flex">{t("getToken.actionButton")}</div>
-						</Button>
-					)}
-				</div>
-				<div className="mt-10 w-2/3">
-					<p className="mb-4">{t("hostURL.subtitle")}</p>
-					<div className="flex w-full">
+			<div className="mt-9 max-w-700">
+				<p className="mb-3 text-base font-light">{t("getToken.subtitle")}</p>
+				{token ? (
+					<div className="flex">
 						<Input
-							aria-label={t("hostURL.copyInputAriaLabel")}
-							className="flex-1 rounded-xl hover:border-gray-1100"
+							aria-label={t("getToken.copyInputAriaLabel")}
+							className="h-12 flex-1 rounded-lg border-gray-950"
 							disabled
-							label={t("hostURL.copyInputAriaLabel")}
-							value={hostURL}
+							label={t("getToken.copyInputAriaLabel")}
+							value={token}
 						/>
 
 						<CopyButton
-							className="ml-4 rounded-md bg-gray-900 px-3 hover:bg-gray-950"
-							successMessage={t("hostURL.copySuccess")}
-							text={hostURL}
+							className="ml-2 gap-2 rounded-md hover:bg-gray-1050"
+							successMessage={t("getToken.copySuccess")}
+							text={token}
 						/>
 					</div>
-				</div>
+				) : (
+					<Button
+						className="h-11 gap-2 border border-gray-750 pl-2.5 pr-3 font-averta font-semibold text-white"
+						onClick={createToken}
+					>
+						{!isLoading ? (
+							<IconSvg alt="New Project" size="xl" src={CircleTokenIcon} />
+						) : (
+							<Loader size="sm" />
+						)}
 
-				<Accordion className="mt-10 w-2/3" title={t("vscodeConfigExample")}>
-					<ImageMotion
-						alt={t("vscodeConfigExample")}
-						src="/assets/image/pages/settings/vscodeConfigurationExample.jpg"
+						{t("getToken.actionButton")}
+					</Button>
+				)}
+			</div>
+			<div className="mt-9 max-w-700">
+				<p className="mb-3 text-base font-light capitalize">{t("hostURL.subtitle")}</p>
+				<div className="flex">
+					<Input
+						aria-label={t("hostURL.copyInputAriaLabel")}
+						className="h-12 flex-1 rounded-lg border-gray-950"
+						disabled
+						label={t("hostURL.copyInputAriaLabel")}
+						value={hostURL}
+					/>
+
+					<CopyButton
+						className="ml-2 gap-2 rounded-md hover:bg-gray-1050"
```

Comment:
Everywhere in the system we use without rounded, so it's a circle. To stay consistent, let's not use `rounded-md` here.
```suggestion
						className="ml-2 gap-2 
 hover:bg-gray-1050"
```

Reply:
changed